website/modules/case-studies-page/services/UrlService.js

```
@@ -0,0 +1,100 @@
+  /**
+   * Builds a case study URL with current query parameters preserved
+   * @param {string} baseUrl - The base URL of the case study
+   * @param {Object} query - Current query parameters from the request
+   * @returns {string} Complete URL with query parameters
+   */
+  static buildCaseStudyUrl(baseUrl, query = {}) {
+    const params = new URLSearchParams();
+
+    // Add industry parameters
+    if (query.industry) {
+      const industries = this.ensureArray(query.industry);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)